src/aws/env.d/development.dist

```diff
@@ -3,16 +3,13 @@
 # AWS admin credentials
 AWS_ACCESS_KEY_ID=yourAwsAccesKeyId
 AWS_SECRET_ACCESS_KEY=YourAwsSecretAccessKey
+AWS_DEFAULT_REGION=eu-west-1
```

Comment:
oh ok. I think this belonged to the previous commit ;-) see the previous comment where I was wondering why removing all mentions of the region.

Reply:
I just move its position because it should be here :-)